src/scripts/background.js

```diff
@@ -682,6 +684,11 @@ window.TogglButton = {
     return timeEntries[timeEntries.length - 1];
   },
 
+  checkDarkMode: async function () {
+    const darkMode = await db.get('darkMode');
+    return darkMode;
```

Comment:
```suggestion
    return darkMode ?? matchMedia('(prefers-color-scheme: dark)').matches;
```
Could we do this? So it defaults to dark-mode if user has enabled it in their system settings

Reply:
:clap: good idea, `??` doesn't work in our current babel config so had to change it a little. I could not actually get this to work in testing though, does it work for you?

Reply:
Ok let's see if this works :thinking: